frontend/hooks/useRewardsHistory.ts

```diff
@@ -147,16 +164,18 @@ type CheckpointsResponse = { checkpoints: CheckpointResponse[] };
  * hook to fetch rewards history for all contracts
  */
 const useContractCheckpoints = () => {
-  const { serviceId } = useServices();
+  const serviceId = useServiceId();
+  const chainId = useChainId();
+  const transformCheckpoints = useTransformCheckpoints();
```

Comment:
this is perpetuating the issue from the previous PR..
if a user has multiple services, this does not work